scripts/backup_db.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
```

Comment:
I'd suggest #!/usr/bin/env bash
in case you installed a newer bash in a non-standard location (/usr/local/bin/bash for example)

Reply:
```suggestion
#!/usr/bin/env bash
```